src/cmd/services/m3query/config/config.go

```diff
@@ -655,7 +568,7 @@ type TagOptionsConfiguration struct {
 	// If not provided, defaults to `le`.
 	BucketName string `yaml:"bucketName"`
 
-	// Scheme determines the default ID generation scheme. Defaults to TypeLegacy.
+	// Scheme determines the default ID generation scheme. Defaults to TypeQuoted.
```

Comment:
Can't comment on the relevant line; but below on :609 (`if cfg.Scheme == models.TypeDefault { //...`), you should set the default scheme to be `Quoted` instead of failing out (also should update the linked doc)